frontend/src/utils/constants.js

```diff
@@ -84,6 +84,19 @@ export const SUPPLEMENTARY_TYPES = Object.freeze({
   TRANSPORT: 3,
 })
 
+export const REQUEST_CATEGORY_TYPES = Object.freeze({
+  ACCOUNT_MAINTENANCE: 'a640abcf-7377-ee11-8179-000d3a09d132',
+})
+
```

Comment:
I asked Anjana and the the entity for that in CRM is migrated from dev to other environments like the entity types that are id: 1,2,3. If that is the case i wonder why they didn't just use 1,2,3,4 ect. Let's get this through for now so QA can start and I'll follow up with you, Viet, Hoang, and can circle back and fix.

Reply:
the request categories id is in the lookup. can we use the id from lookup instead of hardcode it?

Reply:
You've edited my comment "the request categories id is in the lookup. can we use the id from lookup instead of hardcode it?". It seems like it is still hardcoded? we can add this one to app.js:

`getRequestCategoryTypeIdByName: (state) => {
....
    },`

Reply:
Fixed.